Sitecore.Demo.Edge.Website.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16

```

Comment:
@jflheureux - the VS version was updated when saving the solution in VS 2022. It seems to open and build as expected in VS 2019. Let me know if you see any issues.